src/main/java/com/thebois/models/inventory/items/ItemType.java

```diff
@@ -7,17 +7,31 @@
     /**
      * Logs are collected from trees.
      */
-    LOG,
+    LOG(3.0f),
     /**
      * Rocks are collected from stones.
      */
-    ROCK,
+    ROCK(15.f),
```

Comment:
What unit are we imagining this is? Kilos?

Reply:
Like the constructor says, yeah.